src/ORM/Table.php

```diff
@@ -2180,6 +2180,14 @@ public function saveManyOrFail(iterable $entities, $options = []): iterable
      */
     protected function _saveMany(iterable $entities, $options = []): iterable
     {
+        $options = new ArrayObject(
```

Comment:
Does it need to be an ArrayObject?

Reply:
Yes, the options in ORM events are ArrayObject instances.